src/libcephfs.cc

```diff
@@ -982,7 +982,11 @@ extern "C" int ceph_statxat(struct ceph_mount_info *cmount, int dirfd, const cha
 {
   if (!cmount->is_mounted())
     return -CEPHFS_ENOTCONN;
+#if defined(__linux__) && defined(AT_EMPTY_PATH)
+  if (flags & ~(CEPH_REQ_FLAG_MASK|AT_EMPTY_PATH))
```

Comment:
How about defining this flag in `src/include/cephfs/ceph_ll_client.h`? Not sure if it makes sense but couldn't stop myself from commenting

Reply:
Did you mean to define `CEPH_REQ_FLAG_MASK|AT_EMPTY_PATH` as another inside _src/include/cephfs/ceph_ll_client.h_?

Reply:
Yes, as a new statx interface flag

Reply:
Let me know if the updated version goes along with your suggestion.